test/EFCore.PG.FunctionalTests/Query/ArrayArrayQueryTest.cs

```diff
@@ -45,7 +45,7 @@ public void Index_with_constant()
 
             Assert.Single(actual);
             AssertSql(
-                @"SELECT s.""Id"", s.""SomeArray"", s.""SomeByte"", s.""SomeByteArray"", s.""SomeBytea"", s.""SomeMatrix"", s.""SomeText""
+                @"SELECT s.""Id"", s.""SomeArray"", s.""SomeByte"", s.""SomeByteArray"", s.""SomeBytea"", s.""SomeStringArray"", s.""SomeText""
```

Comment:
`SomeMatrix` still exists in the model, but you replaced the columns in all queries by `SomeStringArray` which leads to test failures. What you need is to add the new column and not replace existing:

```suggestion
                @"SELECT s.""Id"", s.""SomeArray"", s.""SomeStringArray"", s.""SomeByte"", s.""SomeByteArray"", s.""SomeBytea"", s.""SomeMatrix"", s.""SomeText""
```